lib/main.dart

```diff
@@ -41,7 +41,8 @@ import './providers/locationStatusProvider.dart';
 import './providers/bluetoothStatusProvider.dart';
 
 //TODO: Implement the coding chanllenge
-
+// 1. Implement a simple UI that informes the user about the collected data
+//    -> for each model do i need to implement a new UI to show the collected data or?
```

Comment:
I don't think for each model a new UI is to be implemented, a single widget that is capable of notifying a user about the data being collected should be enough.